src/api/PlainUpload.js

```diff
@@ -21,15 +21,15 @@ class PlainUpload extends BaseUpload {
      * Handles a upload preflight success response
      *
      * @param {Object} data - Preflight success data
-     * @return {void}
+     * @return {Promise<>}
      */
-    uploadPreflightSuccessHandler = ({ upload_url }: { upload_url: string }) => {
+    uploadPreflightSuccessHandler = async ({ upload_url }: { upload_url: string }): Promise<> => {
```

Comment:
Sync to the latest, flow will complain about <> so will have to change it to something like Promise<any>

Reply:
replace with Promise<*>